packages/usa-icon/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@uswds/usa-icon",
+  "version": "0.0.1-alpha.0",
+  "description": "USWDS icon library",
+  "main": "_index.scss",
+  "scripts": {
+    "test:js": "gulp buildSpriteStandalone && mocha './src/test/*.spec.js'",
```

You can add a `pretest` hook to run a task before testing.

```suggestion
    "pretest": "gulp buildSpriteStandalone",
    "test:js": "mocha './src/test/*.spec.js'",
    "test": "npm run test:js"
```